util/upload-cfn-templates.py

```diff
@@ -38,6 +34,10 @@ def upload_to_s3(args, region):
                     continue
```

Comment:
Don't continue here. If you continue, what happens when 'dryrun' and 'override' both are enabled.

In a case where both dry run and override are enabled, there should be a clear indication of which buckets needoverriding